### ISSUE TITLE
BaseUrl does not allow subdirectory

### ISSUE THREAD

**Issue by hbokh** [vjeantet/hugo-theme-casper#15]
When using `baseurl = "http://example.com/blog"` or  `baseurl = "http://example.com/blog/"`, the links in the HTML are like this: `http://example.com/blogcss/screen.css`.
Please allow subdirectories. Merci!

**Comment by ghreyt** [comment]
I have experienced the same issue and figured out what the problem is.

It is the bug of Hugo v0.13 (https://github.com/spf13/hugo/issues/1105).
.Site.BaseUrl variable in template has trailing slash inconsistently.
They fixed that in v0.14 in two ways:

1. providing the function 'absUrl' for template
1. no trailing slash for .Site.BaseUrl

So simply you can go over this issue with Hugo v0.14 or you can modify a bit of codes like this.
(https://github.com/ghreyt/hugo-theme-casper/commit/e94513c41ee6069682ac44447fb06290043e73e8)

**Comment by hbokh** [comment]
Merci beaucoup! It works as expected with the new hugo 0.14.
I guess you will adapt the theme soon for the new variable and function names, right?
```
ERROR: 2015/05/26 Node's .Url is deprecated and will be removed in Hugo 0.15. Use .URL instead.
ERROR: 2015/05/26 Site's .BaseUrl is deprecated and will be removed in Hugo 0.15. Use .BaseURL instead.
ERROR: 2015/05/26 MenuEntry's .Url is deprecated and will be removed in Hugo 0.15. Use .URL instead.
ERROR: 2015/05/26 Template's safeHtml is deprecated and will be removed in Hugo 0.15. Use safeHTML instead.
```

**Comment by vjeantet** [comment]
fixed with #18

thanks for the update
